### ISSUE TITLE
Angular 2

### ISSUE THREAD

**Issue by adaojunior** [olostan/pretty_samples#2]
Do you have plans to release a angular 2 version of this lib ?

**Comment by olostan** [comment]
Yes. It will be a separate repository. Here is a Angular 2 version: https://github.com/olostan/slides_ng2high/blob/master/lib/sample_directive.dart 

I'll extract and put into pub soon.
